### PR TITLE
Adds basics

### PR DESCRIPTION
**Pull request by iteles**
Adds the very basics of setting yourself up:
+ Get a text editor and set it up with fundamentals
+ Set yourself up on Git and GitHub (with SSH rather than HTTPS)
+ Be comfortable with your terminal setup - added a note here on use of emulators and (non) use of themes. Seemed like the most reasonable place to do it.

Introductory sections still feel too wordy...

Fixes https://github.com/dwyl/start-here/issues/53 by making a start on this.

### CODE REVIEW

**Comment by iteles** [comment]
@nelsonic Please review/comment/amend and merge :+1:

**Comment by nelsonic** [comment]
I'm *not* getting the GitHub notifications... please Message me when ever you assign me a PR. thanks!

**Comment by nelsonic** [comment]
Looks good. :+1:
